data/darktableconfig.xml.in

```diff
@@ -128,6 +128,13 @@
     <longdescription>typically darktable's hand-optimized CPU code is much faster than any OpenCL-on-CPU code; therefore CPUs are excluded from being used as OpenCL devices by default. can be changed by setting this configuration option to TRUE (needs a restart).</longdescription>
   </dtconfig>
   <dtconfig>
+    <name>opencl_disable_drivers_blacklist</name>
+    <type>bool</type>
+    <default>false</default>
+    <shortdescription>whether to allow some partial opencl implementations to be used or not</shortdescription>
+    <longdescription>not all opencl implementations are fully-functional, there are at least two, which claim to be production-quality, which is not true: pocl and beignet; therefore at this point in time, it is better to explicitly blacklist them by default. can be changed by setting this configuration option to TRUE (needs a restart).</longdescription>
```

Comment:
I'd probably phrase it a little bit more politely not really knowing what the developers of beignet and pocl claim or not.

Like: not all opencl implementations are fully-functional, there are at least two, which are not production-quality yet: pocl and beignet; therefore at this point in time, it is better to explicitly blacklist them by default. ...